test/safe-mode-check.js

```diff
@@ -40,13 +39,22 @@ await Promise.all(credentialPaths.map(async credentialPath => {
   const credential = JSON.parse(await fs.promises.readFile(credentialPath));
   try {
     await jsonld.expand(credential, {safe: true});
-    console.log('👍 All terms correctly defined in', credentialPath);
+    if(method !== 'extract') {
+      console.log('👍 All terms correctly defined in', credentialPath);
+    }
   } catch(err) {
-    console.log('😢 Errors found in', credentialPath);
+    if(method !== 'extract') {
+      console.log('😢 Errors found in', credentialPath);
+    }
     console.dir(err, {depth: 5});
     failure = true;
   }
 }));
+
 if(failure) {
   process.exit(1);
 }
+
+if(method === 'extract') {
+  console.dir([...contextUrls].sort());
```

Comment:
What's the intended use of this output?  It's in JavaScript, so difficult to use with automated shell scripting.  Outputting JSON would be easier to pipe to other tools.  But if other more direct tools will be used, maybe this doesn't matter.

Reply:
`npm run extract | grep controller` works fine, do knowing about places where this wouldn't work would help...but I'd prefer we polish the apple over time and merge this as is...pretty please. 😉

Reply:
I'm suggesting this:
```diff
-  console.dir([...contextUrls].sort());
+  console.log(JSON.stringify([...contextUrls].sort(), null, 2));`
```

So one can use the output for basic scripting like:
```sh
for url in `npm -s run extract | jq -r '.[]'`; do a-wonderful-command --url "$url"; done
```

Reply:
The JS array output seems so odd however I think about it.  An alt approach is outputting the lines direct:
```diff
-  console.dir([...contextUrls].sort());
+  console.log([...contextUrls].sort().join('\n'));
```

Reply:
Ah. my bad. I'd forgotten `.dir()` was JS...not JSON...output. That is annoying. I'm going to stick with a JSON array, though, as my expectation has always been that this would be sent into some other system / stored in a document for such a system, and JSON is far simpler to handle than a line delimited text file...weirdly...

Reply:
Fixed in [f3dcf22](https://github.com/credential-handler/vc-examples/pull/83/commits/f3dcf2208ddae55505c3bec6b0d3ceb9a4aa8a6d)